appinterface/projection/crossfire/crossfire_test.go

```diff
@@ -35,11 +39,207 @@ var _ = Describe("Crossfire", func() {
 			1,
 			"foo",
 			"14",
-			"https://example.com/participants.json",
+			"https://raw.githubusercontent.com/foreseaz/random/master/",
```

Comment:
Same above, maybe the full URL ends with JSON is better for config management.

Reply:
https://github.com/crypto-com/chain-indexing/pull/264#discussion_r553123893